### PR TITLE
docs: fix kernel lockdown documentation

### PR DESCRIPTION
**Pull request by arnaldo2792**
**Issue number:**
N/A

**Description of changes:**

```
1c0f4d2d docs: fix kernel lockdown documentation
```

This fixes the kernel lockdown's documentation, since the default values changed in newer variants.

**Testing done:**
Documentation change, no testing required (other than just verify the render).

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -476,7 +476,7 @@ Here are the metrics settings:
 #### Kernel settings
 
 * `settings.kernel.lockdown`: This allows further restrictions on what the Linux kernel will allow, for example preventing the loading of unsigned modules.
-  May be set to "none" (the default), "integrity", or "confidentiality".
+  May be set to "none" (the default in older [variants](variants/)), "integrity" (the default for newer [variants](variants/)), or "confidentiality".
```
Comment (samuelkarp) [inline README.md]: Should we list the specific variants?  Or perhaps just list the variants which are still defaulting to "none"?

Comment (tjkirch) [reply README.md]:
```suggestion
  May be set to "none" (the default in older [variants](variants/), up through aws-k8s-1.19), "integrity" (the default for newer variants), or "confidentiality".
```

Comment (tjkirch) [reply README.md]: Should we make the same change the SECURITY-GUIDANCE.md doc?

Comment (arnaldo2792) [reply README.md]: Yeah, I'll update the Security Guidance with your suggestion :+1:

**Review by webern — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (arnaldo2792) [reply README.md]: I updated both the README and SECURITY-GUIDANCE doc

**Comment by arnaldo2792** [comment]
Forced push includes:

- Rebase
- Added suggestions to both the README.md and SECURITY_GUIDANCE.md docs

**Review by samuelkarp — APPROVED** [review]

**Review by webern — APPROVED** [review]

**Pull request merged**